src/languages/es.ts

```diff
@@ -2395,6 +2395,11 @@ export default {
         copy: 'Copiar',
         copied: '¡Copiado!',
     },
+    actionableWhisperItems: {
+        // Todo: ask for translation
+        invite: 'Invite them',
+        nothing: 'Do nothing',
```

Comment:
I asked for the translations https://expensify.slack.com/archives/C21FRDWCV/p1704166045854639. I'll post them here once someone responds.

Reply:
```js
invite: 'Invitarlos',
nothing: 'No hacer nada'
```